### ISSUE TITLE
support for column resizing

### ISSUE THREAD

**Issue by siddharthlatest** [appbaseio/dejaVu#15]
Support column width resizing to better view data. 

* Long ids are hard to view as the current UI truncates values longer than 20 characters, see https://github.com/appbaseio/dejaVu/issues/11.
* Columns with smaller fields can be compacted and columns with long text data can be expanded to better utilize the screen estate.

The ideal solution should be similar to excel / google sheet's resizing behavior, contributions are welcome :-).

**Comment by agnivade** [comment]
Yes .. this is bothering me too. I have just started looking through the codebase. Most probably I will send a PR soon !
